src/fe/inf_fe_map.C

```diff
@@ -36,34 +37,32 @@ namespace libMesh
 // mapping
 
 
-template <unsigned int Dim, FEFamily T_radial, InfMapType T_map>
-Point InfFE<Dim,T_radial,T_map>::map (const Elem * inf_elem,
-                                      const Point & reference_point)
+Point InfFEMap::map (const unsigned int dim,
+                     const Elem * inf_elem,
+                     const Point & reference_point)
 {
   libmesh_assert(inf_elem);
-  libmesh_assert_not_equal_to (Dim, 0);
+  libmesh_assert_not_equal_to (dim, 0);
 
-  std::unique_ptr<Elem>      base_elem (Base::build_elem (inf_elem));
+  std::unique_ptr<Elem>      base_elem (InfFEBase::build_elem (inf_elem));
 
-  const Order        radial_mapping_order (Radial::mapping_order());
-  const Real         v                    (reference_point(Dim-1));
+  const Order        radial_mapping_order (InfFERadial::mapping_order());
+  const Real         v                    (reference_point(dim-1));
 
   // map in the base face
   Point base_point;
-  switch (Dim)
+  switch (dim)
     {
     case 1:
       base_point = inf_elem->point(0);
       break;
     case 2:
-      base_point = FE<1,LAGRANGE>::map (base_elem.get(), reference_point);
-      break;
     case 3:
-      base_point = FE<2,LAGRANGE>::map (base_elem.get(), reference_point);
+      base_point = FEMap::map (dim, base_elem.get(), reference_point);
```

Comment:
Oh; I found an issue: it must be ``dim-1`` because we are trying to map the base_elem which is the face towards the finite element region.

Reply:
ping @roystgnr ; I am not sure whether notifications are sent for comments in code that is merged already...

Reply:
Thanks!  I'll fix it ASAP.  Any idea why our "Test with complex numbers and infinite elements" CI didn't pick it up?

Reply:
no, not really, I also came across this only by accident, even though I use this function at many places in my code...